commons/src/main/scala/dagr/commons/reflect/ReflectionUtil.scala

```diff
@@ -290,7 +292,11 @@ object ReflectionUtil {
     */
   private[reflect] def typedValueFromString(resultType: Class[_], unitType: Class[_], value: String*): Try[Any] = Try {
     if (ReflectionUtil.isCollectionClass(resultType)) {
-      val typedValues = value.map(v => buildUnitFromString(unitType, v).get).asInstanceOf[Seq[java.lang.Object]]
+      val typedValues = value.indexOf(SpecialEmptyOrNoneToken) match {
```

Comment:
This basically comes from the desire to _add_ the defaults to _overwriting_ the defaults.  It may seem funny that it trickles down to here, but so be it.